### ISSUE TITLE
Feature Idea: Ability to block background <audio> elements?

### ISSUE THREAD

**Issue by AdamF001** [andryou/scriptsafe#174]
Ability to selectively block <audio> elements which play in the background, for sites that have background voices with ads (full pages which cannot be muted unless you have Chrome://Flags enabled or you mute the pc)

**Comment by andryou** [comment]
Thanks for the submission; just to get a better understanding, would this be different from having &lt;audio&gt; blocking enabled?

**Comment by AdamF001** [comment]
Yes, while the block audio in the browser does not disable in-built html <audio> elements (plays without any sort of control no button nothing)
